_health-care/_js/utils/validations.js

```diff
@@ -17,8 +17,13 @@ function isValidPhone(value) {
   return /^\d{3}-\d{3}-\d{4}$/.test(value);
 }
 
+function isValidEmail(value) {
+  return /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(value);
```

Comment:
I want a comment citing the stack overflow logic. :)

The point of documentation is a later dev knows why you made the choice. "I just copied from stack overflow" is super useful info cause the other assumption is "the prior dev though about htis hard and probably had some reason for this...let me check things from first principles again."